src/testRunner/tsconfig.json

```diff
@@ -39,6 +39,7 @@
 
         "unittests/services/extract/helpers.ts",
         "unittests/tsbuild/helpers.ts",
+        "../tsc/executeCommandLine.ts",

```

Comment:
is this location special somehow? It seems weird to add this in the middle of `unittests/*`

Reply:
Didnt want to create project just for it.. This is to include tsc functionality in our tests so its shared. I dont see other way around it

Reply:
Also previously i had copied part of code from tsc.ts into helper and made it to work with System interface.. But it wasnt complete copy. Now it is shared so any change to tsc.ts can be tested to create baselines for that run